Add Project Loved: September 2021 Newspost (#6165)
ppy/osu-wiki: news/2021-09-26-project-loved-september-2021.md

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/389617.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[daisan - -+](https://osu.ppy.sh/beatmapsets/389617#osu) by [Starfy](https://osu.ppy.sh/users/2045914), [-N a n a k o-](https://osu.ppy.sh/users/1407516), [\[ Drop \]](https://osu.ppy.sh/users/2391299), [Chaoslitz](https://osu.ppy.sh/users/3621552), [happy623](https://osu.ppy.sh/users/324768), [IamKwaN](https://osu.ppy.sh/users/1856463), [Kencho](https://osu.ppy.sh/users/3178411), [Kyle Y](https://osu.ppy.sh/users/3565813), [Nakano Itsuki](https://osu.ppy.sh/users/4600383), [Regou](https://osu.ppy.sh/users/419954) and [Rizia](https://osu.ppy.sh/users/1367570)\
185 BPM, 2:05 | 1.10★ – 6.52★\
*description written by [Doomsday](https://osu.ppy.sh/users/18983)*

> Being a massive collaborative effort between many revered mappers in the Hong Kong community from 2015, the mapset for -+ contains a set for every single game mode! Within its full 15 difficulty osu! spread lies intense stream-heavy mapping, with the most extreme difficulties retaining complex cutstream and slider stream patterns at 185 BPM. On top of that, a full suite of custom hitsounding and storyboard effects make this mapset the complete package. You'll be sure to find something to enjoy in this one!

Reviewer: ```suggestion
Being a massive collaborative effort between many revered mappers in the Hong Kong community from 2015, the mapset for *-+* contains a set for every single game mode! Within its full 15 difficulty osu! spread lies intense stream-heavy mapping, with the most extreme difficulties retaining complex cutstream and slider stream patterns at 185 BPM. On top of that, a full suite of custom hitsounding and storyboard effects make this mapset the complete package. You'll be sure to find something to enjoy in this one!
```

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/406332.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[DJ.Kurara - Japanese Transformation](https://osu.ppy.sh/beatmapsets/406332#mania) by [Yuko-](https://osu.ppy.sh/users/2176678)\
260 BPM, 3:44 | 8.41★\
*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*

> Yuko-'s *Japanese Transformation* is probably one of the most iconic 7K rice charts out there. Back when it was uploaded, it was one of the charts that only the legendary ETs could play, and even now, after 5 years, it still presents a formidable challenge to anyone daring to try this monstrosity. At first glance, one can already see the whopping 8.44 star rating - a pretty rare sight, considering this chart contains no long notes. However, patterns used by dreammaster are way more comfortable than they would seem. The most challenging aspect of Japanese Transformation is its bpm, which equals 260 and goes well beyond anything commonly found in the 7K realm. The map starts with a relatively easy intro section, quickly becoming a fury of fast and unforgiving chordstream. Luckily, as mentioned before, the patterning is friendly, which helps manage the blazing pace of the chart, and multiple handtrills help in conserving stamina and HP. After the initial impact, the middle part consists of some basic chordstream and chord smash section, which, albeit still rather difficult, is breathing room for what will come up next. The finale of the chart is the last chorus that contains extended and increasingly dense chordstream with bursts and trills scattered everywhere. All of this comes neatly wrapped up, with every pattern being perfectly understandable, and nothing is out of place. This chart has been executed very well, which is not easy to do at such a difficulty level. That's why advanced 7K players should find this pick very enjoyable and competitive. Vote on this and get yourself a nice place on the leaderboards to assert domination!

Reviewer: ```suggestion
Yuko-'s *Japanese Transformation* is probably one of the most iconic 7K rice charts out there. Back when it was uploaded, it was one of the charts that only the legendary ETs could play, and even now, after 5 years, it still presents a formidable challenge to anyone daring to try this monstrosity. At first glance, one can already see the whopping 8.44 star rating — a pretty rare sight, considering this chart contains no long notes. However, patterns used by dreammaster are way more comfortable than they would seem. The most challenging aspect of Japanese Transformation is its bpm, which equals 260 and goes well beyond anything commonly found in the 7K realm. The map starts with a relatively easy intro section, quickly becoming a fury of fast and unforgiving chordstreams. Luckily, as mentioned before, the patterning is friendly, which helps manage the blazing pace of the chart, and multiple handtrills help in conserving stamina and HP. After the initial impact, the middle part consists of some basic chordstream and chord smash section, which, albeit still rather difficult, is breathing room for what will come up next. The finale of the chart is the last chorus that contains extended and increasingly dense chordstreams with bursts and trills scattered everywhere. All of this comes neatly wrapped up, with every pattern being perfectly understandable, and nothing is out of place. This chart has been executed very well, which is not easy to do at such a difficulty level. That's why advanced 7K players should find this pick very enjoyable and competitive. Vote on this and get yourself a nice place on the leaderboards to assert domination!
```

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/1262927.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Camellia vs. Tsukasa Kaminose - Gunkyouzouteki Soseiriron](https://osu.ppy.sh/beatmapsets/1262927#mania) by [\_Stan](https://osu.ppy.sh/users/1653229), [\[Emiria\]](https://osu.ppy.sh/users/2046044), [biemote](https://osu.ppy.sh/users/799606), [Kyousuke-](https://osu.ppy.sh/users/2616570) and [Penguinosity](https://osu.ppy.sh/users/10235296)\
160 BPM, 4:40 | 5.48★, 5.55★, 5.57★, 5.88★, 6.20★\
*description written by [Penguinosity](https://osu.ppy.sh/users/10235296)*

> \_Stan makes their first appearance to Loved, with their 5 diff collaborative set of Gunkyouzouteki Soseiriron! Stan is a hardworking charter, which I'm sure many people who keep up with ranked charts should be familiar with. From the eclectic unapologetic patterning of \[Dream to Nightmare](https://osu.ppy.sh/beatmapsets/527728#mania/1125576) to the creative and diverse hybrid charting of [Luin of Celtchar](https://osu.ppy.sh/beatmapsets/1281422#mania/2879332), Stan has proven to be an essential member of the community over the years and has even contributed to Ranked standard sets! To any unfamiliar, *Gunkyou* (which I will call the song for short) is a chaotic and intense piano-based track off of Camellia's *Xroniel Xero* album and features a collaboration between him and the equally legendary Tsukasa Kaminose. Onto the set, though, a collaboration between these two artists should be charted as a full collaboration between a myriad of different charters. Those charters include Stan, Kyousuke-, biemote, [Emiria\] (otherwise known as victorica_db), and myself. Besides all of us simply having GDs on the set, the catch is that each of the 5 difficulties is of a different key count. These diffs go from 4K up to 8K, and all of what's here will give players a different experience from one map to the next. From the full rice approach to the 8K to the hybrid centered patterning of the 7 and 4K, each approach to the song brings something distinct to the table. Seeing a well-constructed key spread is rare enough these days, let alone one put together in a way that's been done here. We're both thankful for Stan's contributions to the community ove

Reviewer: > From the full rice approach to the 8K

i don't know what rice is here and thus have no way of checking, but i assume this is supposed to be "From the full rice approach *of* the 8K"

i've also completed the unfinished last sentence using ~~ai autocomplete~~ some slight guesswork

```suggestion
\_Stan makes their first appearance to Loved, with their 5-diff collaborative set of Gunkyouzouteki Soseiriron! Stan is a hardworking charter, which I'm sure many people who keep up with ranked charts should be familiar with. From the eclectic unapologetic patterning of *[Dream to Nightmare](https://osu.ppy.sh/beatmapsets/527728#mania/1125576)* to the creative and diverse hybrid charting of *[Luin of Celtchar](https://osu.ppy.sh/beatmapsets/1281422#mania/2879332)*, Stan has proven to be an essential member of the community over the years and has even contributed to Ranked standard sets! To any unfamiliar, this song is a chaotic and intense piano-based track based on Camellia's *Xroniel Xero* album and features a collaboration between him and the equally legendary Tsukasa Kaminose.

Onto the set, which as a chart for a collaboration between these two artists should be, is a full collaboration between a myriad of different charters. Those charters include Stan, Kyousuke-, biemote, \[Emiria\] (otherwise known as victorica_db), and myself. Besides all of us simply having GDs on the set, the catch is that each of the 5 difficulties is of a different key count. These diffs go from 4K up to 8K, and all of what's here will give players a different experience from one map to the next. From the full rice approach of the 8K to the hybrid centered patterning of the 7K and 4K, each approach to the song brings something distinct to the table. Seeing a well-constructed key-spread is rare enough these days, let alone one put together in a way that's been done here. We're thankful for Stan's contributions to the community over the years and look forward to what they have to offer in the future!
```

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/699875.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Arise - 7K Endurance and technical training pack](https://osu.ppy.sh/beatmapsets/699875#mania) by [Arise](https://osu.ppy.sh/users/5127544)\
144 – 250 BPM, 4:07 | 4.97★ – 7.09★\
The \[Mirror went\] difficulty is *not* being nominated for Loved.\
*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*

> Most of 7K players probably have played a chart from this pack at least once in their o!m career. Arise's *7K Endurance and technical training pack* (also known as Xcaliboor, especially in the 4K community) is a seasoned classic gallery of relatively standard chordstream charts with some LN sections here and there. The choice of difficulties ranges from averagely hard (like Midnight lighting bolt and Sister's noise) to very challenging (Aim burst and Love is war). Although the pack's name mentions technical patterning, its theme is focused on endurance, and there's a tiny technicality in the charts apart from some sparse LN sections. Now a word of warning - patterning of chordstreams used is very basic and doesn't stand out for the most part. The quality of some charts isn't stellar, but the pack never was about following the music as closely as possible. As this is the Endurance pack, its main focus is stamina, and playing these charts is an excellent way to improve the 7K muscles and get used to chordstreaming on many different levels. We hope you will warmly welcome Arise's pack as there is a lot of content to set a score on!

Reviewer: ```suggestion
Most of 7K players probably have played a chart from this pack at least once in their osu!mania career. Arise's *7K Endurance and technical training pack* (also known as Xcaliboor, especially in the 4K community) is a seasoned classic gallery of relatively standard chordstream charts with some LN sections here and there. The choice of difficulties ranges from averagely hard (like "Midnight lighting bolt" and "Sister's noise") to very challenging ("Aim burst" and "Love is war"). Although the pack's name mentions technical patterning, its theme is focused on endurance, and there's a tiny technicality in the charts apart from some sparse LN sections. Now a word of warning: the patterning of chordstreams used is very basic and doesn't stand out for the most part. The quality of some charts isn't stellar, but the pack never was about following the music as closely as possible. As this is the Endurance pack, its main focus is stamina, and playing these charts is an excellent way to improve the 7K muscles and get used to chordstreaming on many different levels. We hope you will warmly welcome Arise's pack as there is a lot of content to set a score on!
```

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/1220647.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[osu!mania 7K Dan Course - LN Dan Phase III](https://osu.ppy.sh/beatmapsets/1220647#mania) by [Jinjin](https://osu.ppy.sh/users/3360737), [\_Kobii](https://osu.ppy.sh/users/6209713), [Kim_GodSSI](https://osu.ppy.sh/users/2218047), [Pengdoll](https://osu.ppy.sh/users/6392061) and [Toaph Daddy](https://osu.ppy.sh/users/7616811)\
167 – 198 BPM, 8:13 | 9.79★, 10.48★, 11.76★, 12.29★, 13.26★\
*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*

> There it is, the final exam of 7K LN abilities and the last part of revamped Jinjin Dans that go up into the Loved voting - *LN Dan Phase III*! There are 5 Dans in this pack: 9th, 10th, Gamma, Azimuth, and Zenith. Since all of you probably know what they are about, I'll keep this short. Dans are sets of 4 songs played consecutively with small breaks inbetween. Each piece is focused on a different aspect of gameplay; in the case of the LN Dans, there are general, tech, inverse, and LN release maps. Passing the course with 95% or more counts as a clear. If you want to know more, be sure to check the mapset's description. These courses are a way to track your progress in the LN department and act as a standard of difficulty for other maps to compare to. Be sure to vote for the last remaining puzzle in the 7K Jinjin Dan set, and good luck with clearing some of the hardest 7K LN courses osu!mania has to offer!

Reviewer: ```suggestion
There it is, the final exam of 7K LN abilities and the last part of revamped Jinjin Dans that go up into the Loved voting — *LN Dan Phase III*! There are 5 Dans in this pack: 9th, 10th, Gamma, Azimuth, and Zenith. Since all of you probably know what they are about, I'll keep this short. Dans are sets of 4 songs played consecutively with small breaks inbetween. Each piece is focused on a different aspect of gameplay; in the case of the LN Dans, there are general, tech, inverse, and LN release maps. Passing the course with 95% or more counts as a clear. If you want to know more, be sure to check the mapset's description. These courses are a way to track your progress in the LN department and act as a standard of difficulty for other maps to compare to. Be sure to vote for the last remaining puzzle in the 7K Jinjin Dan set, and good luck with clearing some of the hardest 7K LN courses osu!mania has to offer!
```

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/389617.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[daisan - -+](https://osu.ppy.sh/beatmapsets/389617#mania) by [hokin1995](https://osu.ppy.sh/users/31054) and [pwhk](https://osu.ppy.sh/users/4887865)\
185 BPM, 2:05 | 1.20★ – 6.44★\
*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*

> hokin1995's 4K difficulties on -+ are rather simple, with a simple chord and jumpstream patterning accompanied with a few LNs and easy bursts. There's nothing much to say about them other than that they're a decent choice for beginner players. On the other hand, 7K difficulties made by pwhk are much more challenging. While lower difficulties resemble the 4K portion of the set, the highest two introduce a lot of complexity to the gameplay. The patterning is based chiefly on chordtrills and doublestreams, some of which are rough in terms of patterning as this map is quite far from modern and resembles BMS style. Some occasional bracket sequences also transition into difficult minijacks on the hardest difficulty. The main meat of the charts is the very fast stair bursts in the middle section followed by an even harder buildup into the somewhat tame chorus, which then again gains on intensity. The chart ends with a dense chordstream and a calm outro. -+ is a pretty fun legacy pick and a remnant of an older era, and it's time it got Loved alongside other game mode difficulties.

Reviewer: ```suggestion
hokin1995's 4K difficulties on *-+* are rather simple, with a simple chord and jumpstream patterning accompanied with a few LNs and easy bursts. There's nothing much to say about them other than that they're a decent choice for beginner players. On the other hand, 7K difficulties made by pwhk are much more challenging. While lower difficulties resemble the 4K portion of the set, the highest two introduce a lot of complexity to the gameplay. The patterning is based chiefly on chordtrills and doublestreams, some of which are rough in terms of patterning as this map is quite far from modern and resembles BMS style. Some occasional bracket sequences also transition into difficult minijacks on the hardest difficulty. The main meat of the charts is the very fast stair bursts in the middle section followed by an even harder buildup into the somewhat tame chorus, which then again gains on intensity. The chart ends with a dense chordstream and a calm outro. *-+* is a pretty fun legacy pick and a remnant of an older era, and it's time it got Loved alongside other game mode difficulties.
```

As usual, we would like to thank everyone who participated in the August votes. You're helping to keep this project alive by doing so, and we appreciate it! If you would like to view the results from the previous round, you may do so by clicking the following links:

- [Results for osu!](https://osu.ppy.sh/community/forums/posts/8269959)
- [Results for osu!taiko](https://osu.ppy.sh/community/forums/posts/8269958)
- [Results for osu!catch](https://osu.ppy.sh/community/forums/posts/8269956)
- [Results for osu!mania](https://osu.ppy.sh/community/forums/posts/8269955)

> September comes with good news for Project Loved! The [Project Loved website](https://loved.sh/) is now fully localisable into your native language. Much thanks to the contributors across the community, the website is available in Spanish, Finnish, Swedish and Turkish for the time being. Interested in translating the website to your native language? If so, head over to the [Localisation page of the Project Loved website](https://loved.sh/localization) and follow the steps there! If you wish to contribute to the Project Loved, feel free to visit the `#osu-loved` channel in the osu!dev Discord server. If website development is your thing, you can follow the ongoing development progress on [the GitHub repository](https://github.com/cl8n/project-loved-web).

Reviewer 2: ```suggestion
September comes with good news for Project Loved! The [Project Loved website](https://loved.sh/) is now fully localisable into your native language. Much thanks to the contributors across the community, the website is available in Spanish, Finnish, Swedish and Turkish for the time being. Interested in translating the website to your native language? If so, head over to the [Localisation page of the Project Loved website](https://loved.sh/localization) and follow the steps there! If you wish to contribute to Project Loved, feel free to visit the `#osu-loved` channel in the [osu!dev Discord server](https://discord.com/invite/ppy). If website development is your thing, you can follow the ongoing development progress on [the GitHub repository](https://github.com/cl8n/project-loved-web).
```

> Before announcing the picks for this month, we would like to introduce you to the new faces of the Project Loved Team and wave goodbyes to those who recently parted ways with us. Already present with us as a metadata reviewer for the picks, [UberFazz](https://osu.ppy.sh/users/8646059), and [Doomsday](https://osu.ppy.sh/users/18983), a long-time osu! player and contributor, have rolled up their sleeves to assist other osu! captains with Loved nominations. Speaking of captains, we are also delighted to present you with [Gamelan4](https://osu.ppy.sh/users/9856910), our newest osu!taiko captain, tasked with picking your favourite dons and kats. Coming from the Global Moderation Team, [Zeus-](https://osu.ppy.sh/users/5464437) takes on the responsibility of beatmap content moderation alongside [Noffy](https://osu.ppy.sh/users/1541323) and [Pachiru](https://osu.ppy.sh/users/2850983). As for our last addition to the Project Loved Team, a renowned Beatmap Nominator, [Hivie](https://osu.ppy.sh/users/14102976) is here to make sure the metadata across all the nominations is accurate.

Reviewer 2: ```suggestion
Before announcing the picks for this month, we would like to introduce you to the new faces of the Project Loved Team and wave goodbyes to those who have recently parted ways with us. Already present with us as a metadata reviewer for the picks, [UberFazz](https://osu.ppy.sh/users/8646059) and [Doomsday](https://osu.ppy.sh/users/18983)—long-time osu! players and contributors—have rolled up their sleeves to assist other osu! captains with Loved nominations. And speaking of captains, we are also delighted to present you with [Gamelan4](https://osu.ppy.sh/users/9856910), our newest osu!taiko captain, tasked with picking your favourite dons and kats. Coming from the Global Moderation Team, [Zeus-](https://osu.ppy.sh/users/5464437) takes on the responsibility of beatmap content moderation alongside [Noffy](https://osu.ppy.sh/users/1541323) and [Pachiru](https://osu.ppy.sh/users/2850983). As for our last addition to the Project Loved Team, renowned Beatmap Nominator [Hivie](https://osu.ppy.sh/users/14102976) is here to make sure the metadata across all the nominations is accurate.
```

> Unfortunately, new arrivals tend to bring along several departures. We have recently parted ways with [X a v y](https://osu.ppy.sh/users/3738344) and [Alter-](https://osu.ppy.sh/users/4980256), osu!taiko and osu!mania captains respectively. Although they are no longer a part of the Project Loved Team, their contributions to the Loved scene are here to stay forever. We would like to thank them for their service and wish them luck in their future endeavours!
> Alright, enough said. Let's get on with this month's nominations:

Reviewer 2: ```suggestion
Unfortunately, new arrivals tend to bring along several departures. We have recently parted ways with [X a v y](https://osu.ppy.sh/users/3738344) and [Alter-](https://osu.ppy.sh/users/4980256), osu!taiko and osu!mania captains respectively. Although they are no longer a part of the Project Loved Team, their contributions to the Loved scene are here to stay forever. We would like to thank them for their service and wish them luck in their future endeavours!

Alright, enough said. Let's get on with this month's nominations.
```
new lines should have one empty line between separating them. this goes for all other occurrences like this

### Navigation

- [osu!](#osu!)
- [osu!taiko](#osu!taiko)
- [osu!catch](#osu!catch)
- [osu!mania](#osu!mania)

## osu!

The osu! Loved candidates were chosen by [Doomsday](https://osu.ppy.sh/users/18983), [UberFazz](https://osu.ppy.sh/users/8646059) and [Wixonater](https://osu.ppy.sh/users/10083084)!

---

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/1204970.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[TJ Davis - Super Sonic Racing](https://osu.ppy.sh/beatmapsets/1204970#osu) by [Woey](https://osu.ppy.sh/users/3792472)\
165 BPM, 3:49 | 6.16★\
*description written by [UberFazz](https://osu.ppy.sh/users/8646059)*

> You know him, you love him, Professor Woey is here to teach us about the wonders of *Super Sonic Racing*. This great track from Sonic R will make you jam so hard you'll barely be able to focus on the game. You'll be sure to love this map if you enjoy playing constant fast sliders and bursts at 165 BPM. *EVERYBODY'S SUPER SONIC RACING, TRY TO KEEP YOUR FEET RIGHT ON THE GROUND!*

Reviewer 2: ```suggestion
You know him, you love him, Professor Woey is here to teach us about the wonders of *Super Sonic Racing*. This great track from [Sonic R](https://en.wikipedia.org/wiki/Sonic_R) will make you jam so hard you'll barely be able to focus on the game. You'll be sure to love this map if you enjoy playing constant fast sliders and bursts at 165 BPM. *EVERYBODY'S SUPER SONIC RACING, TRY TO KEEP YOUR FEET RIGHT ON THE GROUND!*
```

[Merol - Lekker met de meiden](https://osu.ppy.sh/beatmapsets/902007#osu) by [Mijn Aim Zuigt](https://osu.ppy.sh/users/4587487)\
121 BPM, 2:09 | 6.50★, 6.50★\
*description written by [Wixonater](https://osu.ppy.sh/users/10083084)*

What's more fun than hanging out with the ladies? Nothing, well, besides playing a fun Mijn Aim Zuigt beatmap of it.

Unfunny jokes aside, this 6.5\* 121 BPM alternate map lends itself to a lot of fun. It is simply a joy to play with large flowy patterns, fast long sliders, and cross screens to finish them up. With two difficulties, an AR 9.3 and AR 8, you can enjoy reading the map your way, or like Cookiezi's godmode 2 years ago, you could use it for DT. Maybe you want to replicate that or just enjoy fun alt. No matter the reason, I hope you enjoy it!

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/1291269.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[ZUTOMAYO - Haze Haseru Haterumade](https://osu.ppy.sh/beatmapsets/1291269#osu) by [pkk](https://osu.ppy.sh/users/3620525)\
136 BPM, 3:57 | 5.82★\
*description written by [UberFazz](https://osu.ppy.sh/users/8646059)*

> After over 3 and a half years, pkk is back to the Loved section with a song that I'm sure any member of the osu! community can enjoy — *Haze Haseru Haterumade* by ZUTOMAYO! Originally made as a custom map for OWC 2020, this map quickly rose to popularity with its simple yet addictive alt gameplay combined with a fantastic song by a beloved artist. If you love alt maps, give this map a go!

Reviewer 2: ```suggestion
After over 3 and a half years, pkk is back to the Loved section with a song that I'm sure any member of the osu! community can enjoy — *Haze Haseru Haterumade* by ZUTOMAYO! Originally made as a custom map for OWC 2020, this map quickly rose to popularity with its simple yet addictive alt gameplay combined with a fantastic song by a beloved artist. If you love alt maps, give this map a go!
```
```suggestion
After over 3 and a half years, pkk is back to the Loved section with a song that I'm sure any member of the osu! community can enjoy—*Haze Haseru Haterumade* by ZUTOMAYO! Originally made as a custom map for OWC 2020, this map quickly rose to popularity with its simple yet addictive alt gameplay combined with a fantastic song by a beloved artist. If you love alt maps, give this map a go!
```

[daisan - -+](https://osu.ppy.sh/beatmapsets/389617#osu) by [Starfy](https://osu.ppy.sh/users/2045914), [-N a n a k o-](https://osu.ppy.sh/users/1407516), [\[ Drop \]](https://osu.ppy.sh/users/2391299), [Chaoslitz](https://osu.ppy.sh/users/3621552), [happy623](https://osu.ppy.sh/users/324768), [IamKwaN](https://osu.ppy.sh/users/1856463), [Kencho](https://osu.ppy.sh/users/3178411), [Kyle Y](https://osu.ppy.sh/users/3565813), [Nakano Itsuki](https://osu.ppy.sh/users/4600383), [Regou](https://osu.ppy.sh/users/419954) and [Rizia](https://osu.ppy.sh/users/1367570)\
185 BPM, 2:05 | 1.10★ – 6.52★\
*description written by [Doomsday](https://osu.ppy.sh/users/18983)*

Being a massive collaborative effort between many revered mappers in the Hong Kong community from 2015, the mapset for *-+* contains a set for every single game mode! Within its full 15 difficulty osu! spread lies intense stream-heavy mapping, with the most extreme difficulties retaining complex cutstream and slider stream patterns at 185 BPM. On top of that, a full suite of custom hitsounding and storyboard effects make this mapset the complete package. You'll be sure to find something to enjoy in this one!

> Not only does this nomination include the osu! maps, but also the difficulties made for every other game mode in this set! Be sure to read on for the takes of our osu!taiko, osu!catch and osu!mania captains on this colossal project.

Reviewer 2: ```suggestion
Not only does this nomination include the osu! maps, but also the difficulties made for every other game mode in this set! Be sure to read on for the takes on this colossal project from our osu!taiko, osu!catch and osu!mania captains.
```

190.1 BPM, 3:18 | 1.92★, 2.67★, 3.95★\
*description written by [Doomsday](https://osu.ppy.sh/users/18983)*

A time capsule of the early osu! days that was created way back in 2011. This map was originally made as a birthday present to [Larto](https://osu.ppy.sh/users/12328), one of osu!'s most active mappers and community members. Packed with a tremendous fully-illustrated storyboard crammed with osu! lore, and a spread of difficulties in the classic mapping style of Gens, Konbini is an excellent tribute to the first few years of the osu! community. A bona fide piece of osu! history, this is one that every fan of the old-school style should have in their library!

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/720978.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[MiddleIsland - Frailness](https://osu.ppy.sh/beatmapsets/720978#osu) by [LMT](https://osu.ppy.sh/users/7262798)\
150 BPM, 3:12 | 5.97★, 5.97★\
*description written by [Wixonater](https://osu.ppy.sh/users/10083084)*

Dance along! After all, these guitars aren't playing for anyone!

> This medieval battle tune lies at 150 BPM, with more unconventional flow alt patterns and notes snapped mainly to 1/3rd and 1/4th. Containing cut alt patterns into spaced doubles only to be sent back again with a back and forth triple, this map keeps you on your toes, with CS 5 adding another piece to the difficulty. With two diffs for AR 9 and AR 9.5, there is some choice for how you wish to read the map. Play to the vibrant violin! Sing with the songstress! And dance with the guitar! Enjoy!

Reviewer: ```suggestion
This medieval battle tune lies at 150 BPM, with more unconventional flow alt patterns and notes snapped mainly to 1/3 and 1/4. Containing cut alt patterns into spaced doubles only to be sent back again with a back and forth triple, this map keeps you on your toes, with CS 5 adding another piece to the difficulty. With two diffs for AR 9 and AR 9.5, there is some choice for how you wish to read the map. Play to the vibrant violin! Sing with the songstress! And dance with the guitar! Enjoy!
```

In the same light as *[Itte.](https://osu.ppy.sh/beatmapsets/971578#taiko/2034239)* from the July 2021 voting, we've found those breakcore renditions of J-pop / pop-rock songs enjoyable and engaging to play. Therefore, for this round, we brought *Bad for me (Dz'Xa's Amenpunk)* by Official HIGE DANdism!

As expected, KTYN is good at taking these songs and converting them into absolute bangers of a map to ensure it receives the appreciation it deserves. With a map suitable for entry-level Inner Oni players, this is very likely to be one that you'll enjoy playing, as KTYN's style focuses so much on being fun before anything else. Don't miss it!

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/389617.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[daisan - -+](https://osu.ppy.sh/beatmapsets/389617#taiko) by [aabc271](https://osu.ppy.sh/users/155707), [ll-oscar](https://osu.ppy.sh/users/1488736), [Nardoxyribonucleic](https://osu.ppy.sh/users/876419), [Prophecy](https://osu.ppy.sh/users/3793769) and [Skylish](https://osu.ppy.sh/users/2845958)\
185 BPM, 2:05 | 1.44★ – 5.77★\
*description written by [POCARI SWEAT](https://osu.ppy.sh/users/5082685)*

> If you played osu!taiko for a long time, you might have heard the name *-+* at least once. Mapped by 5 veteran osu!taiko mappers from the Hong Kong community, *-+*'s osu!taiko diffs have finally made their way to Loved. There are 8 osu!taiko difficulties mapped with star ratings ranging between 1.44\\~5.77\\*, containing varying patterns and decent BPM structures, with the highest one serving you a pile of 1/6 patterns, albeit relatively subdued. If you're a professional player tired of playing a hell of intricate maps, or a beginner looking for a quality one to practice, this one will be a great choice.

Reviewer: ```suggestion
If you played osu!taiko for a long time, you might have heard the name *-+* at least once. Mapped by 5 veteran osu!taiko mappers from the Hong Kong community, *-+*'s osu!taiko diffs have finally made their way to Loved. There are 8 osu!taiko difficulties mapped with star ratings ranging between 1.44\~5.77\*, containing varying patterns and decent BPM structures, with the highest one serving you a pile of 1/6 patterns, albeit relatively subdued. If you're a professional player tired of playing a hell of intricate maps, or a beginner looking for a quality one to practice, this one will be a great choice.
```

Many of TenyO's maps are pretty challenging, and as the difficulty name of this map might suggest, this is no exception. Tenyo's Fruitcore 2015 (CTB Survival) is a map that will test your osu!catch skills to the max, giving you a variety of patterns ranging from jumps, hyperdash chains, zig-zags, and of course, an all-time favourite, pixels! Now, the hp drain is relatively low, but do not let this fool you. If you miss one of the more packed sections of the map, you'll quickly see that the hp drain feels much higher.

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/832873.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Imagine Dragons - Warriors](https://osu.ppy.sh/beatmapsets/832873#fruits) by [Minato Yukina](https://osu.ppy.sh/users/531253)\
156 BPM, 2:44 | 1.41★ – 7.19★\
*description written by [Sartan](https://osu.ppy.sh/users/4100941)*

Fun fact: 78% of osu! players have played League of Legends at least once in their life! 63% of those people have reached at least level 10, and 84% of those people have reached level 30. Cool right? I made all of that up :). Though one thing I cannot lie about is this map and its quality!

> Once again, we have Minato Yukina with an incredible full set on this beautiful song: Warriors. As you might expect, it is quite a meta-esque beatmap, without any sort of out-of-the-world gimmicks to be found, primarily emphasizing the 1/4ths. Despite this, there is plenty of fun to be had, especially considering this map is generally well-received in the osu! community. The difficulties go up from Bronze to Challenger, so as you might have guessed, there is something to play for everyone, woo!

Reviewer: ```suggestion
Once again, we have Minato Yukina with an incredible full set on this beautiful song: Warriors. As you might expect, it is quite a meta-esque beatmap, without any sort of out-of-the-world gimmicks to be found, primarily emphasizing the 1/4 rhythms. Despite this, there is plenty of fun to be had, especially considering this map is generally well-received in the osu! community. The difficulties go up from Bronze to Challenger, so as you might have guessed, there is something to play for everyone, woo!
```

Anyway, *here we are, don't turn away now*, and go vote!

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/389617.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[daisan - -+](https://osu.ppy.sh/beatmapsets/389617#fruits) by [autofanboy](https://osu.ppy.sh/users/636114)\
185 BPM, 2:03 | 3.91★, 5.45★\
*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*

This mapset was initially nominated for osu!, but it has osu!catch difficulties of good quality on it, so it would be a shame to exclude those. The mapper of these is autofanboy (formerly known as alienflybot), and if you are familiar with their mapping, you know that their maps are usually very clean and have a nice flow. The same goes for these diffs. They are not difficult at all, and most people should be able to FC the Overdose rather quickly. The best feature is the clean, streamy patterns with a fantastic flow. I highly recommend trying them out, for they are super entertaining.

## osu!mania

> The osu!mania Loved candidates were chosen by [Penguinosity](https://osu.ppy.sh/users/10235296)!

Reviewer: ```suggestion
The osu!mania Loved candidates were chosen by [Penguinosity](https://osu.ppy.sh/users/10235296)!

```

With a bit of that history out of the way, let's talk about the map itself. At 210 BPM, you will be experiencing the handstream and roll patterning right out of the gate that Icy has been trademarked for. Honestly, the first couple of minutes can be called pretty tame compared to what the chorus sections offer. Coming off of the more tame intro, the chorus smacks you with an unrelenting, full-density handstream that sees you transitioning from hand to hand with these heavy anchors and splittrills. After that first chorus is completed, DJ Sharpnel was kind enough to lead into a break, and if you ask me, that break is needed for what's to come. The final 2 minutes of the map give you no rest whatsoever as it's a complete splittrill, handstream, and jumptrill hellfest, which has managed to push the stamina and explosivity of players of all kinds for all of these years.

> Keep in mind that this is all just on 1.0 as well. As is tradition with Icy's content and many charts in this vein, rates are provided. We go from 1.0 up to 1.4, which sees the BPM stay at a constant 294, meaning that the 1/8th bursts on 1.4 are approaching nearly 600 BPM. Well, alright, that's just about all I have to say regarding this legendary map. Best of luck to all players, and we once again thank Icy for the vast and imperative content that he has created over these years. Thanks, Icy.

Reviewer: ```suggestion
Keep in mind that this is all just on 1.0 as well. As is tradition with Icy's content and many charts in this vein, rates are provided. We go from 1.0 up to 1.4, which sees the BPM stay at a constant 294, meaning that the 1/8 bursts on 1.4 are approaching nearly 600 BPM. Well, alright, that's just about all I have to say regarding this legendary map. Best of luck to all players, and we once again thank Icy for the vast and imperative content that he has created over these years. Thanks, Icy.
```

*description written by [Penguinosity](https://osu.ppy.sh/users/10235296)*

Evening makes their way back into Loved, with their ever-popular chart of *Hysteric Night Girl*! If you're an osu!mania player who has been in the community for more than a few minutes, you know who Evening is. They're a veteran of the community who has put out some of the most influential and groundbreaking content over these years. From [Singularity](https://osu.ppy.sh/beatmapsets/501530#mania/1073953) and [Backbeat Maniac](https://osu.ppy.sh/beatmapsets/586848#mania/1242696)'s visual spectacles to maps that were once and still used as benchmarks of a player's physical abilities, such as [Tengaku](https://osu.ppy.sh/beatmapsets/411341#mania/892596) or [Meikaruza](https://osu.ppy.sh/beatmapsets/1188943#mania/2477654), Evening has no shortage of meaningful content. With so much content over such a long period, it's easy to look from back then to now and see how their charting style and quirks have developed over the years. One such development is the maturing of Evening's ability to create impactful and fun dump charts such as *Hysteric Night Girl* over here.

> The term "dump charting" can mean many things to a lot of different people, but I'll leave it simple and just say that in the case of this map, Evening chooses to express the sounds in this song through exaggerated methods. In the chorus specifically, you'll find a plethora of stream patterning that constantly shifts between 1/6th and 1/8th snappings, all being tied together by hands. All three choruses employ this structuring; however, they all slightly vary in how they're patterned from each other, which lends to the experience constantly feeling interesting. Outside of that, there's some very tasteful usage of longjacks, LNs, and anchors. It's no wonder why many people have come to enjoy this map, and we love it too. Let's all show our appreciation for what Evening's done over the years by supporting this great map!

Reviewer: seems like i missed this one

```suggestion
The term "dump charting" can mean many things to a lot of different people, but I'll leave it simple and just say that in the case of this map, Evening chooses to express the sounds in this song through exaggerated methods. In the chorus specifically, you'll find a plethora of stream patterning that constantly shifts between 1/6 and 1/8 snappings, all being tied together by hands. All three choruses employ this structuring; however, they all slightly vary in how they're patterned from each other, which lends to the experience constantly feeling interesting. Outside of that, there's some very tasteful usage of longjacks, LNs, and anchors. It's no wonder why many people have come to enjoy this map, and we love it too. Let's all show our appreciation for what Evening's done over the years by supporting this great map!
```

- [Results for osu!](https://osu.ppy.sh/community/forums/posts/8269959)
- [Results for osu!taiko](https://osu.ppy.sh/community/forums/posts/8269958)
- [Results for osu!catch](https://osu.ppy.sh/community/forums/posts/8269956)
- [Results for osu!mania](https://osu.ppy.sh/community/forums/posts/8269955)

September comes with good news for Project Loved! The [Project Loved website](https://loved.sh/) is now fully localisable into your native language. Much thanks to the contributors across the community, the website is available in Spanish, Finnish, Swedish and Turkish for the time being. Interested in translating the website to your native language? If so, head over to the [Localisation page of the Project Loved website](https://loved.sh/localization) and follow the steps there! If you wish to contribute to Project Loved, feel free to visit the `#osu-loved` channel in the [osu!dev Discord server](https://discord.com/invite/ppy). If website development is your thing, you can follow the ongoing development progress on [the GitHub repository](https://github.com/cl8n/project-loved-web).

> Before announcing the picks for this month, we would like to introduce you to the new faces of the Project Loved Team and wave goodbyes to those who have recently parted ways with us. Already present with us as a metadata reviewer for the picks, [UberFazz](https://osu.ppy.sh/users/8646059) and [Doomsday](https://osu.ppy.sh/users/18983)—long-time osu! players and contributors—have rolled up their sleeves to assist other osu! captains with Loved nominations. And speaking of captains, we are also delighted to present you with [Gamelan4](https://osu.ppy.sh/users/9856910), our newest osu!taiko captain, tasked with picking your favourite dons and kats. Coming from the Global Moderation Team, [Zeus-](https://osu.ppy.sh/users/5464437) takes on the responsibility of beatmap content moderation alongside [Noffy](https://osu.ppy.sh/users/1541323) and [Pachiru](https://osu.ppy.sh/users/2850983). As for our last addition to the Project Loved Team, renowned Beatmap Nominator [Hivie](https://osu.ppy.sh/users/14102976) is here to make sure the metadata across all the nominations is accurate.

Reviewer: ```suggestion
Before announcing the picks for this month, we would like to introduce you to the new faces of the Project Loved Team and wave goodbyes to those who have recently parted ways with us. Already present with us as a metadata reviewer for the picks, [UberFazz](https://osu.ppy.sh/users/8646059) and [Doomsday](https://osu.ppy.sh/users/18983) — long-time osu! players and contributors — have rolled up their sleeves to assist other osu! captains with Loved nominations. And speaking of captains, we are also delighted to present you with [Gamelan4](https://osu.ppy.sh/users/9856910), our newest osu!taiko captain, tasked with picking your favourite dons and kats. Coming from the Global Moderation Team, [Zeus-](https://osu.ppy.sh/users/5464437) takes on the responsibility of beatmap content moderation alongside [Noffy](https://osu.ppy.sh/users/1541323) and [Pachiru](https://osu.ppy.sh/users/2850983). As for our last addition to the Project Loved Team, renowned Beatmap Nominator [Hivie](https://osu.ppy.sh/users/14102976) is here to make sure the metadata across all the nominations is accurate.
```

[Merol - Lekker met de meiden](https://osu.ppy.sh/beatmapsets/902007#osu) by [Mijn Aim Zuigt](https://osu.ppy.sh/users/4587487)\
121 BPM, 2:09 | 6.50★, 6.50★\
*description written by [Wixonater](https://osu.ppy.sh/users/10083084)*

What's more fun than hanging out with the ladies? Nothing, well, besides playing a fun Mijn Aim Zuigt beatmap of it.

Unfunny jokes aside, this 6.5\* 121 BPM alternate map lends itself to a lot of fun. It is simply a joy to play with large flowy patterns, fast long sliders, and cross screens to finish them up. With two difficulties, an AR 9.3 and AR 8, you can enjoy reading the map your way, or like Cookiezi's godmode 2 years ago, you could use it for DT. Maybe you want to replicate that or just enjoy fun alt. No matter the reason, I hope you enjoy it!

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/1291269.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[ZUTOMAYO - Haze Haseru Haterumade](https://osu.ppy.sh/beatmapsets/1291269#osu) by [pkk](https://osu.ppy.sh/users/3620525)\
136 BPM, 3:57 | 5.82★\
*description written by [UberFazz](https://osu.ppy.sh/users/8646059)*

> After over 3 and a half years, pkk is back to the Loved section with a song that I'm sure any member of the osu! community can enjoy - *Haze Haseru Haterumade* by ZUTOMAYO! Originally made as a custom map for OWC 2020, this map quickly rose to popularity with its simple yet addictive alt gameplay combined with a fantastic song by a beloved artist. If you love alt maps, give this map a go!

Reviewer: ```suggestion
After over 3 and a half years, pkk is back to the Loved section with a song that I'm sure any member of the osu! community can enjoy — *Haze Haseru Haterumade* by ZUTOMAYO! Originally made as a custom map for OWC 2020, this map quickly rose to popularity with its simple yet addictive alt gameplay combined with a fantastic song by a beloved artist. If you love alt maps, give this map a go!
```

In the same light as *[Itte.](https://osu.ppy.sh/beatmapsets/971578#taiko/2034239)* from the July 2021 voting, we've found those breakcore renditions of J-pop / pop-rock songs enjoyable and engaging to play. Therefore, for this round, we brought *Bad for me (Dz'Xa's Amenpunk)* by Official HIGE DANdism!

As expected, KTYN is good at taking these songs and converting them into absolute bangers of a map to ensure it receives the appreciation it deserves. With a map suitable for entry-level Inner Oni players, this is very likely to be one that you'll enjoy playing, as KTYN's style focuses so much on being fun before anything else. Don't miss it!

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/389617.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[daisan - -+](https://osu.ppy.sh/beatmapsets/389617#taiko) by [aabc271](https://osu.ppy.sh/users/155707), [ll-oscar](https://osu.ppy.sh/users/1488736), [Nardoxyribonucleic](https://osu.ppy.sh/users/876419), [Prophecy](https://osu.ppy.sh/users/3793769) and [Skylish](https://osu.ppy.sh/users/2845958)\
185 BPM, 2:05 | 1.44★ – 5.77★\
*description written by [POCARI SWEAT](https://osu.ppy.sh/users/5082685)*

> If you played osu!taiko for a long time, you might have heard the name *-+* at least once. Mapped by 5 veteran osu!taiko mappers from the Hong Kong community, *-+*'s osu!taiko diffs have finally made their way to Loved. There are 8 osu!taiko difficulties mapped with star ratings ranging between 1.44\\\~5.77\\\*, containing varying patterns and decent BPM structures, with the highest one serving you a pile of 1/6 patterns, albeit relatively subdued. If you're a professional player tired of playing a hell of intricate maps, or a beginner looking for a quality one to practice, this one will be a great choice.

Reviewer: I honestly have no idea what happened here ([twice](https://github.com/ppy/osu-wiki/pull/6165/commits/0beeb90fc2c4ec5c271a3687451aa10bfa9fde33#diff-b3c13c079b9d40eb11b289196e228148f79a999151e594c11802829ba8300020L175-R175) [now](https://github.com/ppy/osu-wiki/pull/6165/commits/b99328e99fe0e751416f42fbb389c1daeaf2933e#diff-b3c13c079b9d40eb11b289196e228148f79a999151e594c11802829ba8300020L175-R175)). Did you look through the diff of your own commit? Did some tool try to automatically escape backslashes?

```suggestion
If you played osu!taiko for a long time, you might have heard the name *-+* at least once. Mapped by 5 veteran osu!taiko mappers from the Hong Kong community, *-+*'s osu!taiko diffs have finally made their way to Loved. There are 8 osu!taiko difficulties mapped with star ratings ranging between 1.44\~5.77\*, containing varying patterns and decent BPM structures, with the highest one serving you a pile of 1/6 patterns, albeit relatively subdued. If you're a professional player tired of playing a hell of intricate maps, or a beginner looking for a quality one to practice, this one will be a great choice.
```

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/808948.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[3R2 - The Truth Never Spoken](https://osu.ppy.sh/beatmapsets/808948#mania) by [\[Crz\]Rachel](https://osu.ppy.sh/users/8369409)\
153 – 198.9 BPM, 2:29 | 6.05★ – 7.64★\
*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*

> Finally, every chordjack enthusiast was waiting for this moment: we're nominating *The Truth Never Spoken* by none other than \\[Crz\\]Rachel! Their chordjack maps gather a lot of popularity, and so is the case with this one — that's why we're happy to put this up, and there will surely be more Rachel mapsets to follow in the future! Now onto the chart itself, this 153 BPM chordjack features relatively light but relentless and draining patterning compressed into two solid sections. Although the density is respectable, the lack of longer anchors allows for some speedjacking strats that make this map viable on higher bpm — that's why rates up to 1.3 were added and HP was decreased. This goes up to almost 200 BPM, which is a challenge for even the best jack gorillas out there, but we're sure every adept jack player will find a good bpm range to practice. The patterns in *The Truth Never Spoken* are clean and comfortable — the main difficulty comes from the bpm itself as well as the length of the runs. What is there more to say? Check out the map, break your fingers, and vote for \\[Crz\\]Rachel's map!

Reviewer: ```suggestion
Finally, every chordjack enthusiast was waiting for this moment: we're nominating *The Truth Never Spoken* by none other than \[Crz\]Rachel! Their chordjack maps gather a lot of popularity, and so is the case with this one — that's why we're happy to put this up, and there will surely be more Rachel mapsets to follow in the future! Now onto the chart itself, this 153 BPM chordjack features relatively light but relentless and draining patterning compressed into two solid sections. Although the density is respectable, the lack of longer anchors allows for some speedjacking strats that make this map viable on higher bpm — that's why rates up to 1.3 were added and HP was decreased. This goes up to almost 200 BPM, which is a challenge for even the best jack gorillas out there, but we're sure every adept jack player will find a good bpm range to practice. The patterns in *The Truth Never Spoken* are clean and comfortable — the main difficulty comes from the bpm itself as well as the length of the runs. What is there more to say? Check out the map, break your fingers, and vote for \[Crz\]Rachel's map!
```

[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/1262927.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Camellia vs. Tsukasa Kaminose - Gunkyouzouteki Soseiriron](https://osu.ppy.sh/beatmapsets/1262927#mania) by [\_Stan](https://osu.ppy.sh/users/1653229), [\[Emiria\]](https://osu.ppy.sh/users/2046044), [biemote](https://osu.ppy.sh/users/799606), [Kyousuke-](https://osu.ppy.sh/users/2616570) and [Penguinosity](https://osu.ppy.sh/users/10235296)\
160 BPM, 4:40 | 5.48★, 5.55★, 5.57★, 5.88★, 6.20★\
*description written by [Penguinosity](https://osu.ppy.sh/users/10235296)*

> \\_Stan makes their first appearance to Loved, with their 5-diff collaborative set of Gunkyouzouteki Soseiriron! Stan is a hardworking charter, which I'm sure many people who keep up with ranked charts should be familiar with. From the eclectic unapologetic patterning of *[Dream to Nightmare](https://osu.ppy.sh/beatmapsets/527728#mania/1125576)* to the creative and diverse hybrid charting of *[Luin of Celtchar](https://osu.ppy.sh/beatmapsets/1281422#mania/2879332)*, Stan has proven to be an essential member of the community over the years and has even contributed to Ranked standard sets! To any unfamiliar, this song is a chaotic and intense piano-based track based on Camellia's *Xroniel Xero* album and features a collaboration between him and the equally legendary Tsukasa Kaminose.

Reviewer: ```suggestion
\_Stan makes their first appearance to Loved, with their 5-diff collaborative set of Gunkyouzouteki Soseiriron! Stan is a hardworking charter, which I'm sure many people who keep up with ranked charts should be familiar with. From the eclectic unapologetic patterning of *[Dream to Nightmare](https://osu.ppy.sh/beatmapsets/527728#mania/1125576)* to the creative and diverse hybrid charting of *[Luin of Celtchar](https://osu.ppy.sh/beatmapsets/1281422#mania/2879332)*, Stan has proven to be an essential member of the community over the years and has even contributed to Ranked standard sets! To any unfamiliar, this song is a chaotic and intense piano-based track based on Camellia's *Xroniel Xero* album and features a collaboration between him and the equally legendary Tsukasa Kaminose.
```

> Onto the set, which as a chart for a collaboration between these two artists should be, is a full collaboration between a myriad of different charters. Those charters include Stan, Kyousuke-, biemote, \\[Emiria\\] (otherwise known as victorica_db), and myself. Besides all of us simply having GDs on the set, the catch is that each of the 5 difficulties is of a different key count. These diffs go from 4K up to 8K, and all of what's here will give players a different experience from one map to the next. From the full rice approach of the 8K to the hybrid centered patterning of the 7K and 4K, each approach to the song brings something distinct to the table. Seeing a well-constructed key-spread is rare enough these days, let alone one put together in a way that's been done here. We're thankful for Stan's contributions to the community over the years and look forward to what they have to offer in the future!

Reviewer: ```suggestion
Onto the set, which as a chart for a collaboration between these two artists should be, is a full collaboration between a myriad of different charters. Those charters include Stan, Kyousuke-, biemote, \[Emiria\] (otherwise known as victorica_db), and myself. Besides all of us simply having GDs on the set, the catch is that each of the 5 difficulties is of a different key count. These diffs go from 4K up to 8K, and all of what's here will give players a different experience from one map to the next. From the full rice approach of the 8K to the hybrid centered patterning of the 7K and 4K, each approach to the song brings something distinct to the table. Seeing a well-constructed key-spread is rare enough these days, let alone one put together in a way that's been done here. We're thankful for Stan's contributions to the community over the years and look forward to what they have to offer in the future!
```
